spec.html

```diff
@@ -128,8 +128,17 @@ <h1>Conformance</h1>
   <p>A conforming implementation of ECMAScript must not implement any extension that is listed as a Forbidden Extension in subclause <emu-xref href="#sec-forbidden-extensions"></emu-xref>.</p>
   <p>A conforming implementation of ECMAScript must not redefine any facilities that are not implementation-defined, implementation-approximated, or host-defined.</p>
   <p>A conforming implementation of ECMAScript may choose to implement or not implement <dfn>Normative Optional</dfn> subclauses. If any Normative Optional behaviour is implemented, all of the behaviour in the containing Normative Optional clause must be implemented. A Normative Optional clause is denoted in this specification with the words "Normative Optional" in a coloured box, as shown below.</p>
-  <emu-clause id="sec-conformance.normative-optional" example normative-optional>
-    <h1>Example Clause Heading</h1>
+  <emu-clause id="sec-conformance-normative-optional" oldids="sec-conformance.normative-optional" example normative-optional>
+    <h1>Example Normative Optional Clause Heading</h1>
+    <p>Example clause contents.</p>
+  </emu-clause>
+  <p>A conforming implementation of ECMAScript must implement Legacy subclauses, unless they are also marked as Normative Optional. All of the language features and behaviours specified within Legacy subclauses have one or more undesirable characteristics. However, their continued usage in existing applications prevents their removal from this specification. These features are not considered part of the core ECMAScript language. Programmers should not use or assume the existence of these features and behaviours when writing new ECMAScript code.</p>
```

Comment:
I thought the plan was to keep the `<dfn>` on `Legacy`, so that it would link in the clause attribute heading.

Reply:
Ah, that hadn’t been added yet. I’ll do so tonight.